app/models/application_record.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  # protect_from_forgery with: :exception
```

Comment:
I wasn't sure if this was where this goes or not. I was just following the pets app, so that's why it's commented out at the moment.